package.json

```diff
@@ -91,6 +91,7 @@
     "lodash": "^4.17.10",
     "prop-types": "^15.6.1",
     "react-custom-scrollbars": "^4.2.1",
+    "react-docgen-typescript": "^1.12.0",
```

Comment:
Oops, this should be in `devDependencies`

Reply:
Thanks for the catch! Moved.